packages/node-resolve/test/types.ts

```diff
@@ -12,13 +12,10 @@ const config = {
   plugins: [
     nodeResolve({
       browser: true,
-      customResolveOptions: {
-        moduleDirectory: 'js_modules'
-      },
+      moduleDirectory: 'js_modules',
       dedupe: ['lodash'],
       extensions: ['.mjs', '.js', '.jsx', '.json'],
       jail: '/my/jail/path',
-      only: ['some_module', /^@some_scope\/.*$/],
       preferBuiltins: false,
```

Comment:
This one appears to miss the `preserveSymlinks` option as well. Might be a good time to double-check this file, the TypeScript types, the documentation and the actual code are in sync.

Reply:
There is no option on the plugin itself anymore for this. Added others that were missing 👍